src/visibility.js

```diff
@@ -184,6 +167,12 @@ class Visibility extends BasePlugin {
     this.eventManager.listen(this.player, this.player.Event.LEAVE_PICTURE_IN_PICTURE, () => {
       this._isInPIP = false;
     });
+    this._viewablityManager = new Utils.ViewabilityManager();
+    this._viewablityManager.observe(this._appTargetContainer, this.player.config.playback.visibilityThreshold / 100, (visible: boolean) => {
```

Comment:
I think we should consider opening a new config cause it's not so relevant to playback

Reply:
You use here the same parameters to observe as in playkit why don't you just listen to playkit event?
if you want to be able to observe another element with a different threshold from this plugin I would use it like that

Reply:
It is not the same. In Kaltura i observe config.ui.targetId (The player itself - even after floating)
Here we observe config.targetId (The placeholder)